README.md

```diff
@@ -215,6 +223,204 @@ objectives:
     that is used as the denominator. Received data is used to compare objectives
     (threshold) values to find total number of metrics.
 
+#### Alert Policy
+
+An Alert Policy allows you to define the alert conditions for a SLO.
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertPolicy
+metadata:
+  name: string
+  displayName: string # optional
+spec:
+  description: string # optional
+  conditions: # list of alert conditions
+    - conditionRef: # required when alert condition is not inlined
+  notificationTargets:
+  - targetRef: # required when alert notification target is not inlined
+
+```
+
+#### Notes (Alert Policy)
+
+- **conditions\[ \]** *Alert Condition*, required field.
+  A condition can be inline defined or can refer to external Alert condition defined
+  in this case the following are required:
+  - **conditionRef** *string*: this is the name or path the Alert condition
+- **notificationTargets\[ \]** *Alert Notification Target*, required field.
+  A condition can be inline defined or can refer to external Alert Notification Target
+  defined in this case the following are required:
+  - **targetRef** *string*: this is the name or path the Alert Notification Target
+
+An example of a Alert policy which refers to another Alert Condition:
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertPolicy
+metadata:
+  name: AlertPolicy
+  displayName: Alert Policy
+spec:
+  description: Alert policy for cpu usage breaches, notifies on-call devops via email
+  conditions:
+    - conditionRef: cpu-usage-breach
+  notificationTargets:
+    - targetRef: OnCallDevopsMailNotification
+
+```
+
+---
+
+#### Alert Condition
+
+An Alert Condition allows you to define in which conditions a alert of SLO
+needs to be triggered.
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertCondition
```

Comment:
I think that this might be better to merge in with with the alert policies, as opposed to its own kind

Reply:
Originally, I had the idea of being able to define multiple conditions and reuse them in alert policies. Guess, this made make it all a bit too unnecessary difficult/complex?

Reply:
yah, i think that might be a bit too much. lets just merge them

Reply:
Makes sense